analyzers/ruby/ruby_test.go

```diff
@@ -0,0 +1,41 @@
+package ruby_test
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/fossas/fossa-cli/analyzers/ruby"
+	"github.com/fossas/fossa-cli/module"
+	"github.com/fossas/fossa-cli/pkg"
+	"github.com/stretchr/testify/assert"
```

Comment:
[nit] Separate import groups of third-party and first-party code. In general, import group ordering (top-to-bottom) is standard library, third party, first-party.